client/landing/stepper/declarative-flow/internals/steps-repository/site-migration-instructions/steps/step-add-migration-key.tsx

```diff
@@ -0,0 +1,43 @@
+import clsx from 'clsx';
```

Comment:
The only real change here is https://github.com/Automattic/wp-calypso/pull/92950/files#r1690471450.

The rest of this component didn't change. It was just extracted from `useSteps`.